scripts/scrapers-us-municipal-crontask

```diff
@@ -1,4 +1,7 @@
 # /etc/cron.d/scrapers-us-municipal-crontask
+APPDIR=/home/datamade/scrapers-us-municipal
```

Comment:
:applause:

Reply:
re: your question, i think these variables are pretty self-explanatory!

Reply:
i'd say go ahead and use them for nyc and chicago, as well, if you'd like.

Reply:
All right! Done.